File: instructor Henry/lenguajes/Ruby/act-27-05-2024.rb
puts "escriba el numero de renta"
R = gets.chomp.to_f

if R <= 10
    renta1 = (R * (20 / 100.to_f))
    renta = R - renta1
    puts("El valor final de la renta es: #{renta}$")
elsif R <= 100
    renta1 = (R * (10 / 100.to_f))
    renta = R - renta1
    puts("El valor final de la renta es: #{renta}$")
elsif  R <= 1000
    renta1 = (R * (5 / 100.to_f)) 
    renta = R - renta1
    puts("El valor final de la renta es de: #{renta}$")
else 
    renta1 =  (R * (1 / 100.to_f)) 
    renta = R - renta1
    puts("El valor final de la renta es de: #{renta}$")

end

File: instructor Henry/lenguajes/Dart/act-27-05-2024.dart
import 'dart:io';

void main(){
  print('Escriba el número de declaración de renta');
  int R = int.parse(stdin.readLineSync()!);

  if (R <= 10){
    double renta1 = (R * 20 / 100);
    double renta = R - renta1;
    print('El valor final de la renta es: $renta\$');
  }else if (R <= 100){
    double renta1 = (R * 10 / 100); 
    double renta = R - renta1;
    print('El valor final de la renta es: $renta\$');
  }else if (R <= 1000){
    double renta1 = (R * 5 / 100);
    double renta = R - renta1; 
    print('El valor final de la renta es: $renta\$');
  }else {
    double renta1 = (R * 1 / 100); 
    double renta = R - renta1;
    print('El valor final de la renta es: $renta\$');
  }
}